letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+   if integer < 60
+    "F"
+  elsif integer < 70
+    "D"
+  elsif integer < 80
+    "C"
+  elsif integer < 90
+    "B"
+  else
+    "A"
+  end
```

Comment:
Case statement might be more appropriate once the `elsif`s start stacking up, but this is fine.